examples/ars.js

```diff
@@ -14,16 +14,16 @@ function remove_intermediate($, selector) {
     $(selector).each((i, elem) => $(elem).replaceWith($(elem).contents()));
 }
 
-function ingest_article(hatch, {title, description, url, created}) {
-    return Libingester.util.fetch_html(url).then($ => {
-        const BASE_URI = Libingester.util.get_doc_base_uri($, url);
+function ingest_article(hatch, entry) {
```

Comment:
Fair but most of the vars were used once so it seemed like it was unnecessary. I'll change it if you want.